.github/workflows/test_gzcz.yml

```diff
@@ -0,0 +1,23 @@
+name: testing gzcz
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 13.2.0
+      - uses: ottocho/gzcz@master
+        with:
+          setup_command: npm install
+          build_command: npm run build
+          dist_path: 'dist'
+          app_name: 'szzz'
+          storage_type: 'gcs'
+          storage_bucket: 'test'
+          github_token: ${{ GITHUB_TOKEN }}
```

Comment:
I am not sure if this env works like this. maybe `${{ secrets.GITHUB_TOKEN }}` ?